.github/workflows/linux.yml

```diff
@@ -1268,7 +1268,7 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores'}}
+    runs-on: ubuntu-20.04-16-cores
```

Comment:
Do you expect that execution time will be reduced in twice? The more powerful runner costs twice as much

Reply:
I will return runner, I just want to see if model passes. On slower runner it couldn't finish